src/components/organisms/dashboard/projectsTableRow.tsx

```diff
@@ -0,0 +1,192 @@
+import React, { MouseEvent, KeyboardEvent } from "react";
+
+import moment from "moment";
+import { useTranslation } from "react-i18next";
+import { NavigateFunction } from "react-router-dom";
+
+import { dateTimeFormat } from "@src/constants";
+import { DeploymentStateVariant, SessionStateType } from "@src/enums";
+import { SidebarHrefMenu } from "@src/enums/components";
+import { DashboardProjectWithStats } from "@src/types/models";
+import { cn, getSessionStateColor } from "@src/utilities";
+
+import { IconButton, IconSvg, StatusBadge, Td, Tr } from "@components/atoms";
+
+import { ActionStoppedIcon, ExportIcon, TrashIcon } from "@assets/image/icons";
+
+export const DashboardProjectsTableRow = ({
+	id,
+	name,
+	status,
+	totalDeployments,
+	running,
+	stopped,
+	completed,
+	error,
+	lastDeployed,
+	deploymentId,
+	handelDeactivateDeployment,
+	downloadProjectExport,
+	displayDeleteModal,
+	navigate,
+}: DashboardProjectWithStats & {
+	displayDeleteModal: (
+		status: DeploymentStateVariant,
+		deploymentId: string,
+		projectId: string,
+		projectName: string
+	) => void;
+	downloadProjectExport: (projectId: string) => void;
+	handelDeactivateDeployment: (deploymentId: string) => Promise<void>;
+	navigate: NavigateFunction;
+}) => {
```

Comment:
If we already optimizing the components and making them smaller let's take this definition to a separate type/interface:
```
import { NavigateFunction } from "react-router-dom";
import { DeploymentStateVariant } from "@src/enums";
import { DashboardProjectWithStats } from "@src/types/models";

export interface DashboardProjectsTableRowProps extends DashboardProjectWithStats {
  handlers: {
    displayDeleteModal: (
      status: DeploymentStateVariant,
      deploymentId: string,
      projectId: string,
      projectName: string
    ) => void;
    downloadProjectExport: (projectId: string) => void;
    handelDeactivateDeployment: (deploymentId: string) => Promise<void>;
    navigate: NavigateFunction;
  };
}
```